include/circt/Dialect/FIRRTL/FIRRTLCanonicalization.td

```diff
@@ -89,20 +89,20 @@ def EmptyNode : Pat<
   [(EmptyAttrDict $annotations)]>;
 
 // regreset(clock, invalidvalue, resetValue) -> reg(clock)
-def RegresetWithInvalidReset : Pat<
+def RegResetWithInvalidReset : Pat<
   (RegResetOp $clock, (InvalidValueOp), $_, $name, $annotations),
   (RegOp $clock, $name, $annotations),
   []>;
 
 // regreset(clock, reset, invalidvalue) -> reg(clock)
 // This is handled by the `RemoveReset` pass in the original Scala code.
-def RegresetWithInvalidResetValue : Pat<
+def RegResetWithInvalidResetValue : Pat<
   (RegResetOp $clock, $_, (InvalidValueOp), $name, $annotations),
   (RegOp $clock, $name, $annotations),
   []>;
 
 // regreset(clock, constant_zero, resetValue) -> reg(clock)
-def RegresetWithZeroReset : Pat<
+def RegResetWithZeroReset : Pat<
```

Comment:
I don't know what is the right capitalization here. 🤔  But, feel free to land this separately if it makes the change here cleaner.

Reply:
I don't know that there is a "right" answer, but please be consistent with other spellings.

Reply:
It looks like Fabian already switched this over to `RegReset` over `Regreset`. I'll keep the fix as it aligns with the name of the op in CPP, i.e., `RegResetOp`.